hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -1,6 +1,6 @@
 from hailtop.utils import (partition, url_basename, url_join, url_scheme,
                            url_and_params, parse_docker_image_reference, grouped)
-from hailtop.utils.utils import unzip
+from hailtop.utils.utils import (unzip, flatten)
```

Comment:
Can you remove the parantheses? You don't need them here because there aren't that many imports that you'd need multiple lines.

Reply:
Removed parantheses